app/uk/gov/hmrc/fileupload/controllers/EnvelopeController.scala

```diff
@@ -50,21 +48,24 @@ class EnvelopeController(withBasicAuth: BasicAuth,
 
   def create() = Action.async(jsonBodyParser[CreateEnvelopeRequest]) { implicit request =>
     def envelopeLocation = (id: EnvelopeId) => LOCATION -> s"${ request.host }${ uk.gov.hmrc.fileupload.controllers.routes.EnvelopeController.show(id) }"
-    val command = CreateEnvelope(nextId(), request.body.callbackUrl, request.body.expiryDate, request.body.metadata)
+    val command = CreateEnvelope(nextId(), request.body.callbackUrl, request.body.expiryDate, request.body.metadata, CreateEnvelopeRequest.constraintsWithDefaultsIfNotProvided(request.body.constraints, envelopeDefaultConstraints))
```

Comment:
I am not sure about that. What are the facts here? The defaults at this moment might be also and important fact.